docs/how_to/openedx/release_upgrades.md

```diff
@@ -0,0 +1,131 @@
+# How To Update The Release Version Of An Open edX Deployment
+
+## Open edX Releases
+Open edX cuts new named releases approximately every 6 months. Each of our deployments
+needs to be able to independently upgrade the respective version on their own
+schedule. MITx Residential upgrades twice per year, roughly aligned with the release
+timing of the upstream Open edX platform, whereas xPro has typically upgraded on an
+annual basis in the December timeframe.
+
+## Open Learning's Open edX Deployments
+At Open Learning we have four distinct installations of Open edX, each with their own
+release cadence and configuration requirements. For each of those deployments we have
+multiple environment stages (e.g. CI, QA, Production). Each of those deployment * stages
+combinations needs to be able to have their versions managed independently, with newer
+versions being progressively toggled to higher environments.
```

Comment:
I don't understand what "toggled" means in this context.

Reply:
"Propagated" is probably a better term. I'll wordsmith this a bit for better clarity.